chevah/github_hooks_server/handler.py

```diff
@@ -56,6 +47,33 @@ def ping(self, event):
             return 'Pong! But GitHub Zen text is missing.'
         return f'Pong! {zen}'
 
+    def pull_request(self, event):
+        """
+        Called, among other cases, when a PR review is requested.
+
+        Triggers the same actions as the needs-review command to issue_comment.
+
+        The `review_requested` action is under the `pull_request` event:
+        https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
+        """
+        action = event.content.get('action')
+        if action != 'review_requested':
+            logging.warning(f"No handler for pull_request action '{action}'.")
```

Comment:
I think that this should be logged as debug.
We might have many other normal actions for a PR, like "created, assigned or closed"

And maybe also include `ready_for_review` . This is when a PR is no longer a draft.

Reply:
`ready_for_review` would indeed be nice! No need to fumble about selecting someone to request a review from, if you already typed them in the description.

Then, they appear automatically in the "Reviewers" list after they reviewed.